src/Sylius/Bundle/AttributeBundle/Resources/config/services.xml

```diff
@@ -25,6 +30,19 @@
             <tag name="doctrine.event_subscriber" />
             <argument>%sylius.attribute.subjects%</argument>
         </service>
+
+        <service id="sylius.registry.attribute_type" class="%sylius.registry.attribute_type.class%">
+            <argument>Sylius\Component\Attribute\AttributeType\AttributeTypeInterface</argument>
+        </service>
+        <service id="sylius.form.type.attribute_type_choice" class="%sylius.form.type.attribute_type_choice.class%">
+            <argument>%sylius.attribute.attribute_types%</argument>
+            <tag name="form.type" alias="sylius_attribute_type_choice" />
+        </service>
+
+        <service id="sylius.attribute.validator.attribute" class="%sylius.attribute.validator.attribute.class%">
+            <argument type="service" id="sylius.registry.attribute_type" />
+            <tag name="validator.constraint_validator" alias="attribute_value_validator" />
```

Comment:
I think alias should include ``sylius_`` prefix. ``sylius_valid_attribute_value_validator``?